core/src/main/java/io/questdb/cairo/wal/ApplyWal2TableJob.java

```diff
@@ -568,6 +570,14 @@ private int processWalCommit(
                 mvRefreshTask.operation = MatViewRefreshTask.INVALIDATE;
                 mvRefreshTask.invalidationReason = "truncate operation";
                 return 1;
+            case MAT_VIEW_INVALIDATE:
+                long txn_old = writer.getTxn();
```

Comment:
should be simply

```java
writer.markSeqTxnCommitted(seqTxn);
return 1;
```

without any if

Reply:
fixed in: https://github.com/questdb/questdb/pull/5507/commits/18dd32480f52eef2f3aac2b537db27d64cbf1cee